plugins/org.eclipse.papyrus.uml.diagram.sequence.figure/src/org/eclipse/papyrus/uml/diagram/sequence/figure/LifelineBodyFigure.java

```diff
@@ -31,6 +31,8 @@
 
 	private static final int TOLERANCE = 5;
 
+	private int anchorHeight;
```

Comment:
The `AnchorParser` API uses *distance* terminology instead of height, which makes sense because an anchor is a point object and so doesn't have height.  And this looks like it serves as a default value for the anchor parameter, so perhaps it should be named `defaultAnchorDistance`?  Similarly the get/set API then.

Reply:
Agreed